frontend/packages/kubevirt-plugin/src/utils/index.ts

```diff
@@ -14,7 +14,7 @@ import {
   getKind,
   getUID,
 } from '@console/shared/src/selectors';
-import { VM_TEMPLATE_NAME_PARAMETER } from '../constants/vm-templates';
+import { getRandomChars } from '@console/shared/src/utils';
```

Comment:
```suggestion
import { getRandomChars } from '@console/shared/src/utils/utils';
```

this should fix the cyclic dependencies. We should try to avoid importing from index files as much as possible.

note: other solution could be to move `generateDataVolumeName` function to standalone file to break the deps. But have not tried if it works.

Reply:
That worked! I can't believe I missed it!